app/javascript/helpers/storage_manager/load-provider-capabilities.js

```diff
@@ -12,4 +14,19 @@ const loadProviderCapabilities = (providerId) => API.get(`/api/providers/${provi
     return options;
   });
 
-export default loadProviderCapabilities;
+const parseCapabilitiesForPhysical = async(providerCapabilities, id) =>
+  API.get(`/api/physical_storage_families/${id}?attributes=capabilities`)
+    .then(({ capabilities }) => {
+      const valueArray = [];
+      Object.keys(capabilities).forEach((capabilityName) => {
+        capabilities[capabilityName].forEach((capabilityValue) => {
+          valueArray.push({
+            label: sprintf(__('%s: %s'), capabilityName, capabilityValue),
```

Comment:
Are `capabilityName` and `capabilityValue` strings we need to give to the translation team? Or are they user defined values. That will determine wether or not we need to translate them or not. `CapabilityName` at least feels like it shouldn't be translated as you don't want to translate names typically.

Reply:
This function is used to iterate thru the capabilities (names and values) and build the valueArray with the label, value structure needed for the multi select component in the form. 
(uuid's fetch and search for in the provider capabilities list as @Fryguy ask from us in our former conversation )

Reply:
the user is supposed to see "compression: True", "compression: False", "thin_provision: True", etc.
is that something MIQ is supposed to translate?

Reply:
If the options for `capabilityValue` are just true / false we do need to translate them. For `capabilityName` are the strings "compression" and "thin_provision" user defined? If yes we don't need to translate them.